tests/Table.test.jsx

```diff
@@ -207,3 +216,73 @@ describe("Table", () => {
     });
   });
 });
```

Comment:
Please add test cases for the visibility of the delete and hide options. For instance, the delete option should be visible only if `isDeletable` is present, and the hide option should be visible by default but not visible if `isHidable` is false.